general/macros.hpc

```diff
@@ -373,13 +373,16 @@ cat << EOF > %{buildroot}%{hpc_module_dep_base}%{version}
 # -N  <pname>: Specify if a sub-package has different basename (pname)
 # -q: For internal use only!
 # -M: use specified name in place of %%name
-# -O: Obsoleted package with version %version, multiple obsoletes may be
-#     separated by ":"
+# -O  <fullname>: Add Provides/Obsoletes for package <fullname> with 
+#     version %version, multiple names may be separated by ":"
+# -P  <prefix>: Add Provides/Obsoletes with %version-%release for package 
+#     names derived from the master package name, replacing <prefix> with
+#     the contents of %<prefix>_provides. Used for primary python3 packages.
```

Comment:
The %<prefix>_provides seems to be something python specific. Thus, the -P argument only makes sense in a python context. 
In other contexts this variable may not be provided by some external context. In order to use the -P option the potential user would need to define %<prefix>_provides to use this option. Thus, it should at least be documented in more detail what is expected and what the result will be. An example would be nice.